src/Social/Models/SystemModules.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Kanvas\Packages\Social\Models;
+
+use Canvas\Models\SystemModules as KanvasSystemModules;
+use Phalcon\Mvc\ModelInterface;
+use Phalcon\Di;
+
+class SystemModules extends KanvasSystemModules
+{
+    /**
+     * Initialize method for model.
+     */
+    public function initialize()
+    {
+        parent::initialize();
+    }
+
+    /**
+     * Get System Module by its model_name.
+     *
+     * @deprecated v2
+     *
+     * @param string $model_name
+     *
+     * @return ModelInterface
+     */
+    public static function getSystemModuleByModelName(string $modelName) : ModelInterface
```

Comment:
@rwhite27 doesnt kanvas core have this already?

Reply:
Had to override because of the cache element.